docs/roadmap.md

```diff
@@ -9,97 +9,49 @@ You may obtain a copy of the License at
 
 ## Security
-
--   Access Control on Stream operation
--   Encrypted connections among Clients, Controller and SegmentStore
--   Encrypted connections among Controller and SegmentStore, and ZK and HDFS
-
-## Direct Access to Streams in HDFS
-
-Allow Hadoop MapReduce jobs to access stream data stored in Tier 2 HDFS.
-
-## Low Level Reader API
-
--   A more flexible way to coordinate readers in a group with changing number of segments. 
-
-## Connectors for Stream Processors
-
-Additional streaming connectors allow data ingestion into Flink and using more
-systems as sinks for Flink streaming programs.
-
--   Spark (source/sink)
--   Apache Beam (TBD)
--   (TBD)
-
-# Version 0.4 (Sep 2017)
-
-## Object Storage as a Tier 2 Storage
-
-Ability to use Object storage as Tier 2 storage Layer. Preferably Amazon S3 or
-any storage systems that support S3 API. 
-
-## Pravega Mesos Framework
-
-To fully support dynamic scaling of Pravega cluster based on resource
-utilization. This would entail that we build Pravega framework that negotiates
-with Mesos Resource Manager on securing or releasing additional resource. 
-
-# Future Items - Not versioned yet. Subject to discussion 
-
--   REST Proxy for Reader/Writer (REST proxy for Admin operations is already there)
--   Stream aliasing
--   Stream trimming and retention
--   Ability to Geo-Replicate events
--   Ability to logically group multiple Streams
--   Exposing information for administration purposes
--   Provide default Failure Detector
--   Ability to assign arbitrary Key-Value pairs to streams - Tagging
--   Non-disruptive and rolling upgrades for Pravega
--   Policy driven tiering of Streams from Streaming Storage to Long-term storage
--   Ability to track/share schema of Stream message payload (Schema Registry or
-    equivalent)
--   Stream consumption/read of a Stream from multiple sites via the Streaming
-    API, HDFS and NFS APIs
--   Ability to enable reads/writes to/from Streams across all data-centers
-    (Active-Active)
--   Logical isolation of streams provisioned by different tenants
--   Ability for Operator to define soft quotas for tenants
--   Ability to define QoS for tenants (min MB/s per stream segment guarantees)
--   Kafka API Compatibility (Producer and Consumer APIs)
+Security for this release will focus on support for securing Pravega external interfaces along with basic access controls on stream access and administration.
+-  Access Control on Stream operations
+-  Auth between Clients and Controller/SegmentStore
+-  Auth between SegmentStore and Tier 2 Storage
+
+## Pravega Connectors
+Pravega ecosystem interconnectivity will be augmented with the following:
+-  Expanded Flink connector support (batch & table API support)
+-  Logstash connector
+-  Other (TBD)
```

Comment:
TBD sounds like we forgot to update. Should we say something like others under development? Perhaps we should also add a note inviting others to contribute, like "interested in writing a connector? talk to us...".